lilli-hart/2025-06.md

```diff
@@ -1,5 +1,67 @@
 # Weeknotes: June 2025
 
+## w/c 30 June 2025
+
+### Inbox tracker
+
+* 💌 (Kirstie to show me!)
```

Comment:
Yes! Happy to!

But also - I bet this isn't really useful for you 😂  Feel free to delete if you're already at the Inbox Zero zen master level 😂